db/seeds.rb

```diff
@@ -719,7 +719,7 @@
 # Store -------------------------------------------------------------------------
 puts 'Store'
 
-StoreItem.create({ name: '9 Volt Battery', price: 1, quantity: 1000})
+StoreItem.create({ name: '9 Volt Battery', price: 0.99, quantity: 1000})
```

Comment:
Don't change the store item prices in the seeds file.  If you need to verify it works, change the objects in the test seeds.  This file is currently used to persist long-term data (store prices, tool barcodes, etc.) so we don't want to make changes like this.